ansible/README.md

```diff
@@ -33,13 +99,16 @@ There are a few variables that may need user configuration and they are found in
 | `install_microshift` | Build and install MicroShift from git | `false` |
 | `build_etcd_binary` | Build and deploy a separate etcd process | `false` |
 
-## Running
+### Inventory Configuration
 
-There are a few places to configure before execution.
+The inventory configuration file is located at `inventory/inventory`.
 
-As with any Ansible playbook, the first file to configure is the inventory located at `inventory/inventory`.
+The MicroShift host name or IP address must be configured to match the current environment. If the user account available on the MicroShift server is not `microshift`, it should be changed as well.
 
-### Inventory Configuration
+> The following Ansible command line options can be used instead of editing the `inventory/inventory` file.
```

Comment:
I don't think we should move away from the inventory file as I mentioned above.

Reply:
I'm trying to avoid editing files in the repository for the main benchmark flows. It makes everything much easier.

Reply:
If there's a problem that I'm missing, let's try to make it work.

Reply:
The problem is there's group/host specific variables that we need to set.

Reply:
I see. I'll change the code to set group variable only.

Reply:
I changed this to ansible_host_var and ansible_user_var - is this better now?